print_deps.rb

```diff
@@ -15,8 +17,37 @@ def initialize(name, depends)
   end
 end
 
+def cache_dir
+  name = File.expand_path("../cache", __FILE__)
```

Comment:
so if script in future will live in /usr/bin, then it create stuff there? I hope not.